packages/formatters/src/formatters.ts

```diff
@@ -373,6 +373,87 @@ export function messageLink<C extends Snowflake, M extends Snowflake, G extends
 	return `${guildId === undefined ? channelLink(channelId) : channelLink(channelId, guildId)}/${messageId}`;
 }
 
+export enum HeadingLevel {
+	One = 1,
+	Two,
+	Three,
+}
+
+/**
+ * Formats the content into a heading level one.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param content - The content to wrap
+ */
+export function heading<C extends string>(content: C, level?: HeadingLevel.One): `# ${C}`;
+
+/**
+ * Formats the content into a heading level two.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param content - The content to wrap
+ */
+export function heading<C extends string>(content: C, level: HeadingLevel.Two): `## ${C}`;
+
+/**
+ * Formats the content into a heading level three.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param content - The content to wrap
+ */
+export function heading<C extends string>(content: C, level: HeadingLevel.Three): `### ${C}`;
+
+export function heading(content: string, level?: HeadingLevel) {
+	switch (level) {
+		case HeadingLevel.Three:
+			return `### ${content}`;
+		case HeadingLevel.Two:
+			return `## ${content} `;
+		default:
+			return `# ${content}`;
+	}
+}
```

Comment:
so bit of a weird suggestion but
```suggestion
export function heading(content: string, level?: HeadingLevel) {
	return `${'#'.repeat(level)} ${content}`;
}
```

not really required though.

Reply:
1. it works for more 
2. not efficient

Reply:
Yeah, don't think that's needed

Reply:
yeah just me being weird

Reply:
I see where you're coming from, and it'd be a good idea in some cases, not in this case though